examples/save_window.rs

```diff
@@ -0,0 +1,75 @@
+/// Example of saving window position to enable "hot reloading".
```

Comment:
its a very niche and super platform-specific example, maybe keep it in some repo outside of miniquad itself? there might be a separate examples repo linked from `awesome-quads` list or something

Reply:
That's fair. Removed.